OracleWebLogic/samples/12213-webtier-apache/weblogic.conf

```diff
@@ -1,7 +1,7 @@
-# Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2018-2020 Oracle and/or its affiliates.
 #
-# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
-LoadModule weblogic_module /root/lib/${MOD_WLS_PLUGIN}
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+LoadModule weblogic_module /u01/oracle/lib/mod_wl_24.so
```

Comment:
The old version uses MOD_WLS_PLUGIN env var. Is there a reason to change it to the hard-coded value?